sdk/include/reactos/wine/winioctl.h

```diff
@@ -433,7 +433,7 @@ typedef struct _FILE_FS_DEVICE_INFORMATION {
 
 /* FileFsAttributeInformation = 5 */
 typedef struct _FILE_FS_ATTRIBUTE_INFORMATION {
-	ULONG	FileSystemAttribute;
+	ULONG	FileSystemAttributes;
 	LONG	MaximumComponentNameLength;
```

Comment:
Note to /self: review and fix the type of this 2nd member in our ndk and xdk.

Reply:
That's already correct: https://github.com/search?q=repo%3Areactos%2Freactos%20_FILE_FS_ATTRIBUTE_INFORMATION&type=code

Reply:
This winioctl.h, and MSDN, documents this second member as: `LONG MaximumComponentNameLength;`
while we have:
https://github.com/reactos/reactos/blob/d663eb446689525b376f0fd447fd6eb5f2da3698/sdk/include/ndk/iotypes.h#L835-L838
and:
https://github.com/reactos/reactos/blob/d663eb446689525b376f0fd447fd6eb5f2da3698/sdk/include/xdk/iotypes.h#L6005-L6007
marked as `ULONG MaximumComponentNameLength;`